### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by dr4hcu5-jan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,25 @@
...
-      wisdom-oss.isService: true
-      wisdom-oss.service.name: someName
-      wisdom-oss.service.path: /somePath
-      wisdom-oss.service.upstream-name: someUpstream
-```
\ No newline at end of file
+> [!CAUTION]
+> Changing the OpenID Connect configuration in the watchdog only will result
+> in a locked platform since the frontend is unable to receive the required
+> values dynamically. Remember to rebuild the frontend after changing the values
```
Comment (cptpiepmatz) [inline README.md]:
```suggestion
> values dynamically. Remember to rebuild the frontend after changing the values.
```

File: init.go
Comment (cptpiepmatz) [inline init.go:1]: I expect fatal errors to destroy the process. Why are we not exiting here?

File: resources/environment.json
Comment (cptpiepmatz) [inline resources/environment.json:1]: Why are removing the json5? The top comment was quite nice.

File: init.go
Comment (dr4hcu5-jan) [reply init.go:1]: as documented in the [official docs](https://pkg.go.dev/log#Fatal) for the log package calling `log.Fatal()` automatically calls `os.Exit(1)` resolving in an automatic exit indicating that something went wrong

File: resources/environment.json
Comment (dr4hcu5-jan) [reply resources/environment.json:1]: The json5 has been replaced to unify the resources in each service

**Review by cptpiepmatz — APPROVED** [review]

**Review by cptpiepmatz — APPROVED** [review]

**Pull request merged**